extensions/interactions/GraphInput/GraphInput.js

```diff
@@ -269,6 +269,7 @@ oppia.directive('graphViz', function() {
               y: $scope.state.mouseY,
               label: ''
             });
+            setMode0(_MODES.MOVE);
```

Comment:
In that case, should setMode be called onClickSomething and setModeHandler be called setMode?